gomock/call.go

```diff
@@ -288,7 +294,8 @@ func (c *Call) isPreReq(other *Call) bool {
 }
 
 // After declares that the call may only match after preReq has been exhausted.
-func (c *Call) After(preReq *Call) *Call {
+func (c *Call) After(prq interface{GetCall() *Call}) *Call {
```

Comment:
It would be nice if this interface could be an exported type of this package. That would allow users to do the following:
```go
var calls []gomock.GetCaller
// programmatically append expected calls to the calls slice
gomock.InOrder(calls...)
```

See https://github.com/quic-go/quic-go/pull/4057/files#diff-53170cac6baf0bf02f3aaa3994259ea4cffaaf2498e768fb7e03cf9f752af96cL1413.